esi_leap/api/controllers/v1/contract.py

```diff
@@ -54,9 +54,12 @@ class ContractsController(rest.RestController):
     def get_one(self, contract_uuid):
         request = pecan.request.context
         cdict = request.to_policy_values()
-        policy.authorize('esi_leap:contract:get', cdict, cdict)
+        policy.authorize('esi_leap:contract:list', cdict, cdict)
```

Comment:
It's weird to rename the policy check for fetching one contract to "list" instead of "get". If you think we only need one of the two, it should probably be 'get'.

Reply:
So the two policies would be get and list_al? Or just get and list?

Reply:
The reason I did this is because I don't see why the listing policy would be different than the policy for getting just one contract

Reply:
Yep, it's definitely fair to just use one! But if you just use one, name it 'get' instead of 'list'.

Reply:
so the two policies will be get and list_all? or should it be get and list

Reply:
just slightly confused

Reply:
It should either be 'get' or 'list', or you could just use 'get' and remove 'list' completely.

Reply:
In other words, if you're just using one policy rule, replace `esi_leap:contract:list` everywhere with `esi_leap:contract:get`

Reply:
With one policy, then anybody can just see all of the contracts? I thought we didn't want that

Reply:
The way things are coded, its hardcoded that if you want to allow everybody to see any contract, then they can't because of checks like request.project != project_id

Reply:
I will remedy this by calling the policy rather than just raising the error at that point

Reply:
Wasn't your change already just having one policy, using 'list'? I'm just suggesting that you use 'get' instead of 'list'.

Reply:
We currently have two policies for contracts

Reply:
list_all for admins, and list for owners and lessees

Reply:
Whoops - what I meant is, instead of using 'list' and 'list_all', use 'get' and 'get_all'.

Reply:
And perhaps rename `get_all` to `get_admin`?